h/static/scripts/group-forms/components/GroupModeration.tsx

```diff
@@ -1,11 +1,48 @@
+import { Spinner } from '@hypothesis/frontend-shared';
 import { useState } from 'preact/hooks';
 
 import type { Group } from '../config';
+import { useGroupAnnotations } from '../hooks/use-group-annotations';
 import GroupFormHeader from './GroupFormHeader';
 import type { ModerationStatus } from './ModerationStatusSelect';
 import ModerationStatusSelect from './ModerationStatusSelect';
 import FormContainer from './forms/FormContainer';
 
+function AnnotationsList({
+  filterStatus,
+}: {
+  filterStatus?: ModerationStatus;
+}) {
+  const { loading, annotations } = useGroupAnnotations({ filterStatus });
+
+  if (loading) {
+    return (
+      <div className="mx-auto">
+        <Spinner size="md" />
+      </div>
+    );
+  }
+
+  if (annotations.length === 0) {
+    return (
+      <div
+        className="border rounded p-2 text-center"
+        data-testid="annotations-fallback-message"
+      >
+        {filterStatus === 'PENDING'
+          ? 'You are all set!'
```

Comment:
Is this a placeholder or was it taken from a mock?

Reply:
Mostly a placeholder, as there are no mocks for loading states or empty lists. We can bring this up with the broader team.

Reply:
I have opened a discussion to decide what should be displayed https://hypothes-is.slack.com/archives/C2C2U40LW/p1747818733428699